common/src/main/java/io/druid/metadata/MetadataStorageConnectorConfig.java

```diff
@@ -38,8 +38,8 @@
   @JsonProperty
   private String user = null;
 
-  @JsonProperty
-  private String password = null;
+  @JsonProperty("password")
```

Comment:
This is completely transparent to the user. We could add docs about the fact that there are password providers, but that might be more confusing than anything, given that we don't ship any other ones, I don't think this is necessary until we offer other password providers.